bus.go

```diff
@@ -91,22 +90,12 @@ func toSubscription[MessageType proto.Message](sub subInternal, size int) Subscr
 	}
 }
 
-type nilSubscription[MessageType any] struct {
-	closeOnce sync.Once
-	c         chan MessageType
-}
-
-func (s *nilSubscription[MessageType]) Channel() <-chan MessageType {
-	return s.c
-}
+type nilSubscription[MessageType any] struct{}
 
-func (s *nilSubscription[MessageType]) Close() error {
-	s.closeOnce.Do(func() { close(s.c) })
+func (s nilSubscription[MessageType]) Channel() <-chan MessageType {
 	return nil
 }
 
-func SubscribeNil[MessageType proto.Message]() Subscription[MessageType] {
-	return &nilSubscription[MessageType]{
-		c: make(chan MessageType),
-	}
+func (s nilSubscription[MessageType]) Close() error {
+	return nil
 }
```

Comment:
it might be useful to be able to close these. i suppose we can add it later if there's an actual use case though 🤔